docs/user-docs/latest/settings/managing-global-settings.md

```diff
@@ -10,10 +15,37 @@ Single Sign-On (SSO) can be made available on a Strapi application to allow admi
 
 To configure the SSO feature settings:
 
-1. Go to the *Global settings > Single Sign-On* area of the settings interface.
+1. Go to the *Global settings > Single Sign-On* sub-section of the settings interface.
 2. Define your chosen new settings:
 
 | Setting name      | Instructions                                                                                                                                                                                                                                                        |
 | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | Auto-registration | Click on **ON** to allow the automatic creation of a new Strapi administrator when an SSO login does not match an existing Strapi administrator account. If this setting is set on **OFF**, new Strapi administrators accounts must be created manually beforehand. |
 | Default role      | Choose among the drop-down list the role to attribute by default to auto-registrated Strapi administrators through SSO login.                                                                                                                                       |
+
+## Configuring Internationalization locales
+
+The [Internationalization plugin](/user-docs/latest/plugins/strapi-plugins.md#internationalization-plugin) allows to manage content in different languages, called "locales". Once the Internationalization plugin is installed in a Strapi application (see [Installing plugins via the Marketplace](../plugins/installing-plugins-via-marketplace.md)), administrators can manage locales from the *Global settings > Internationalization* sub-section of the settings interface.
+
+The *Internationalization* settings sub-section displays a table listing all locales available for the Strapi application. By default, only the English locale is configured and set as the default locale. 
+
+For each locale, the table displays the default ISO code of the locale, its optional display name and indicates if the locale is set as the default one. From the table, administrators can also:
```

Comment:
In the dev docs, Pierre added a link for the ISO stuff, maybe it's worth it to add it there too?

Reply:
Yes I already added a link to the full list of locales in "Global settings > Configuring Internationalization locales"